http-native/src/main/java/org/ballerinalang/net/http/HttpUtil.java

```diff
@@ -1788,6 +1791,11 @@ public static String encodeString(String value) {
         }
     }
 
+    static StrandMetadata getStrandMetadata(String method) {
+        return new StrandMetadata(BALLERINA_BUILTIN_PKG_PREFIX, PROTOCOL_HTTP, HttpPackageUtil.getHttpPackageVersion(),
```

Comment:
Can we make these in init function similar to io package

Reply:
This will create strand meta data for each request

Reply:
Which init method?? `HttpPackageUtil.getHttpPackageVersion()`??

Reply:
Done